docs/overview/multiparty/multiparty_flow.md

```diff
@@ -73,10 +118,10 @@ In the _deterministic compute_ section we talked about the value that determinis
 of multi-party logic can have. Either through on-chain execution, or advanced privacy preserving
```

Comment:
We list deterministic compute here, but its listed lower on the side bar so it's possible the reader hasn't looked at this section yet. Perhaps the easiest thing is to just link to the page? 
![Screen Shot 2022-09-15 at 12 27 55 PM](https://user-images.githubusercontent.com/10987380/190457757-adeeb66b-1e90-40aa-b741-8a25825a719f.png)

Reply:
Re-did this section